### ISSUE TITLE
Support for strftime type of functionality

### ISSUE THREAD

**Issue by jplauri** [vaexio/vaex#679]
With pandas, one can easily get a desired string representation from an underlying date type:

```
import pandas as pd
import numpy as np

dates = pd.date_range("01-01-2016", "14-04-2020", freq="6H")
amount = np.random.randint(1, 5000, len(dates))

x = pd.DataFrame({"date" : dates, "amount" : amount})
# Some suitable date format
date_format = "%Y-%m"
x["date"].dt.strftime(date_format)
```

To my best understanding, there is no similar support for this in vaex. If that's indeed the case, could such functionality be added?

FWIW, I'd be happy to try to contribute here. I'm currently trying to figure out the codebase as to where the related functionality is sitting at.

**Comment by jplauri** [comment]
I'm not at all familiar with the codebase and not that experienced with the library to begin with, but would a reasonable implementation simply add into [vaex-core/vaex/functions.py](https://github.com/vaexio/vaex/blob/master/packages/vaex-core/vaex/functions.py) the following (comments omitted)?

```
@register_function(scope='dt', as_property=True)
def dt_strftime(x):
    import pandas as pd
    return pd.Series(_pandas_dt_fix(x)).dt.strftime(format)
```

I'm not sure as to what `as_property=True` actually designates.

**Comment by JovanVeljanoski** [comment]
Hi @jplauri 

This looks good to me. Would you mind opening a PR for this?  Perhaps also a unit-test, adding to the `test_datetime_operations()` in datetime_test.py ?

Thank you!

**Comment by jplauri** [comment]
@JovanVeljanoski would be more than happy to! Let me get back to this over the weekend if not later today.

**Comment by maartenbreddels** [comment]
Solved in #682 and already released in alpha versions of vaex-core 2
